server/index.js

```diff
@@ -69,20 +69,60 @@ app.get('/user/find', (req, res) => {
 	});
 });
 
+let waitingUser = {}
+
+const matchMaker = () => {
+	setInterval(() => {
+		if(userModule.getWaitingUserLen() > 1){
+			let keys = userModule.getWaitingUserKeys()
+			let roomval = userModule.getWaitingUserKeys()[0] + userModule.getWaitingUserKeys()[1]
+			userModule.getWaitingUser(keys[0]).join(roomval)
+			userModule.getWaitingUser(keys[1]).join(roomval)
```

Comment:
you didn't let it go randomly?

hmmm

Reply:
It will be small code to get random users from existing list. I just did to test we can do it randomly as well.

Reply:
> It will be small code to get random users from existing list. I just did to test we can do it randomly as well.

okay makes sense

Reply:
Fixed this added random picking user code in user.js file